src/components/Dapp.re

```diff
@@ -28,6 +29,35 @@ module EditButton = {
   };
 };
 
+module Streak = {
+  [@react.component]
+  let make = (~gorilla: Gorilla.gorilla) => {
+    let gorillaName = Gorilla.getName(gorilla);
+
+    let daysHeld =
+      PureHooks.useTimeAcquiredGorilla(gorilla)
+      ->Belt.Option.map(dateAquired =>
+          MomentRe.diff(MomentRe.momentNow(), dateAquired, `days)
+        );
+
+    switch (daysHeld) {
+    | Some(daysHeldFloat) =>
+      let numDaysStr = daysHeldFloat->Js.Float.toFixed;
+
+      <Rimble.Tooltip
+        message={j|$gorillaName has been held for $numDaysStr days by the same owner.|j}
+        placement="top">
```

Comment:
Think this message can have more of a postive light. 
Whooooop "@   " has held "gorilla" for "x" days! Streak 🥇

Reply:
Yeah, makes sense. I'll see what I can do :)